EIPS/eip-831.md

```diff
@@ -20,27 +18,25 @@ URIs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages prov
 
 ### Syntax
 
-Ethereum URIs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+Ethereum URIs contain "ethereum" or "eth" in their schema (protocol) part and are constructed as follows:
 
-    request                 = "ethereum" ":" [ prefix "-" ] payload
+    request                 = "eth" [ "ereum" ] ":" [ prefix "-" ] payload
     prefix                  = STRING
     payload                 = STRING
 
 ### Semantics
 
-`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility to ERC-67. When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
-
-`payload` is mandatory and the content depends on the prefix. Structuring of the content is defined in the ERC for the specific use-case and not in the scope of this document. One example is ERC-681 for the pay- prefix.
+`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility to a former URL format by @alexvandesande (#67) that was commonly used but never formalized and so cannot be mentioned by it's name without making the EIPW bot angry. When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
```

Comment:
```suggestion
`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility with a common pattern used prior to the introduction of this standard. When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
```

Reply:
Thanks - but it was never used on "Ethereum mainnet" as this is used on another layer and not really on a network - and seems I can reference EIP-67 again soon as for: https://github.com/ethereum/EIPs/pull/5338
This kinda solves it.

Reply:
*IF* that gets approved by the author you can reference EIP-67.  Note that this PR will be blocked from merging until then though, and be prepared for that to never happen as many EIP authors never respond.  😢